modules/ztp-image-based-upgrade-generate-seed-image.adoc

```diff
@@ -0,0 +1,221 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-seed-generation_{context}"]
+= Generating a seed image with the {lcao}
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` CR. The Operator does the following:
+
+// seed cluster, some things are cluster specific that are not included
+// all operator, LCAO, OADP, LVMS, included in the seed image, telco day2 stuff is incl
+// system config checks ()
+
+* Checks for required system configurations
+* Performs any necessary system cleanup before generating the seed image
+* Launches the image generation task, which involves the following:
+** Stops cluster operators
+** Prepares the seed image configuration
+** Generates and pushes the seed image to the image repository specified in the `ImageBasedUpgrade` CR
+** Restores cluster operators
+** Expires seed cluster certificates
+* Generates new certificates for the seed cluster
+* Restores and updates the `SeedGenerator` CR on the seed cluster, and the `ManagedCluster` CR on the hub cluster
+
+//TODO Expand on what the seed image contains and explicitly say what it doesn't
+
+.Prerequisites
+
+* You deployed a {sno} with DU profile.
+* You installed the {lcao} on the seed cluster.
+* You installed the OADP Operator on the seed cluster.
+* You are logged in as a user with `cluster-admin` privileges.
+* The seed cluster has the same CPU topology as the target cluster.
+* The seed cluster is registered as a managed cluster.
+* The {lcao} deployed on the target cluster is compatible with the version in the seed image.
+// Are there more prereqs?
+
+.Procedure
+
+. Detach the seed cluster from the hub cluster either manually or if using ZTP, by removing the `SiteConfig` CR from the `kustomization.yaml`.
+
+.. To manually detach the seed cluster, run the following command:
++
+[source,terminal]
+----
+$ oc delete managedcluster sno-worker-example
+----
+
+.. Wait until the `ManagedCluster` CR is removed. Once the CR is removed, create the proper `SeedGenerator` CR. The {lcao} cleans up the {rh-rhacm} artifacts.
+
+.. If you are using GitOps ZTP, remove the seed cluster's `SiteConfig` CR from the `kustomization.yaml`:
++
+[source,yaml]
+----
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+generators:
+#- example-seed-sno.yaml
+- example-target-sno1.yaml
+- example-target-sno1.yaml
+----
+
+.. Commit the `kustomization.yaml` changes in your Git repository and push the changes.
++
+The ArgoCD pipeline detects the changes and removes the managed cluster.
+
+. Create the `Secret` CR in the `openshift-lifecycle-agent` namespace.
++
```

Comment:
I found the step alignment a bit confusing here. Should it be:
1. Create the secret 
 a. Step 1 create a yaml file etc.
 b. create auth
 c.  Copy the secret.
 d. apply the secret CR
 2. Next task..